src/blockdata/script.rs

```diff
@@ -805,15 +805,14 @@ mod test {
     }
 
     #[test]
-    #[cfg(all(feature = "serde", feature = "strason"))]
+    #[cfg(feature = "serde")]
```

Comment:
Needs to be `feature = "serde_json"`, no?

Reply:
No, you're testing the serde stuff in the main code and serde_json is a dev-dependency that is always present for testing.

(I was mistaken in the Cargo file, though, I forgot to move serde_json to dev-dependencies. We don't rely on it for anything else than tests.)